backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/PartnerController.java

```diff
@@ -370,4 +370,19 @@
         }
         return HttpResponse.serverError();
     }
+
+    /**
+     * Aries: Send message to partner
+     *
+     * @param id  {@link UUID} the partner id
+     * @param msg {@link SendMessageRequest}
+     * @return HTTP status
+     */
+    @Post("/{id}/message")
+    public HttpResponse<Void> requestCredential(
```

Comment:
requestCredential?

Reply:
ha, good catch. Must've had something else on my mind when I did this!  Will change this.